src/views/ModalContainer.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FullscreenModal } from '@edx/paragon';
+
+/* The purpose of this component is to wrap views with a header/footer for situations
+ * where we need to run non-embedded
+ */
+
+const ModalContainer = ({ Component, title }) => (
+  <FullscreenModal
+    isOpen
+    onClose={null}
+    hasCloseButton={false}
+    title={title}
+    modalBodyClassName="content-body"
```

Comment:
I have been using modal `footerNode` attribute. We probably need to do css work around to have footer as children instead.